src/cliapp.rs

```diff
@@ -26,10 +26,94 @@ pub fn make_app() -> App<'static, 'static> {
                 .global(true)
                 .help("The path to the config folder."),
         )
-        .subcommand(App::new("run").about("Run the relay").after_help(
-            "This runs the relay in the foreground until it's shut down.  It will bind \
+        .subcommand(
+            App::new("run")
+                .about("Run the relay")
+                .after_help(
+                    "This runs the relay in the foreground until it's shut down.  It will bind \
              to the port and network interface configured in the config file.",
-        ))
+                )
+                .arg(
+                    Arg::with_name("secret_key")
```

Comment:
The names here are with underscores, and in all other cases with dashes. Can we unify this?
I personally prefer underscores for the name.

Reply:
Ok, I prefer the names to be the same as the parameters, in this case they are with underscore to keep it identical with the code used in the credentials sub command. Also the name is only internal to the code **but you do have a point, it should be consistent**.
I'll change all internal names to use underscores.